internal/bminventory/inventory_v2_handlers.go

```diff
@@ -600,6 +600,21 @@ func (b *bareMetalInventory) validateIgnoredValidations(problems []string, ignor
 	if err != nil {
 		problems = append(problems, fmt.Sprintf("error during unmarshal of json for ignored %s validations", validationType))
 	} else {
+		for _, v := range ignoredValidationsArr {
+			var err error
+			if validationType == "cluster" {
```

Comment:
nit: could this "magic value" be declared as constant  (also in the calling code)? Same for "host"